09-mapping.qmd

```diff
@@ -64,12 +64,17 @@ tm_shape(nz) +
 ## Interactive maps
 
 - When are interactive maps useful
+An interactive map is an important way to understand and interpret complex geographical information. A good interactive map enables movement 
```

Comment:
Another comment on the 'house style': sentence based not max line width breaks pls.

Reply:
I've managed to break my GitHub repository and will start again from a clean PR to address this.